src/source/src/parser/protobuf_parser.rs

```diff
@@ -20,14 +20,14 @@ use risingwave_common::array::Op;
 use risingwave_common::error::ErrorCode::{self, InternalError, ItemNotFound, ProtocolError};
 use risingwave_common::error::{Result, RwError};
 use risingwave_common::types::{DataType, Datum, Decimal, OrderedF32, OrderedF64, ScalarImpl};
+use risingwave_expr::vector_op::cast::str_to_date;
```

Comment:
Better not to import `risingwave_expr` for `risingwave_source`. We have `use super::common::str_to_date;`

Reply:
The function is duplicated, so I deleted `super::common::str_to_date`.

Reply:
Consider the compiling time, we duplicate the `str_to_date` and `str_to_timestamp` functions here.